src/nsidc/metgen/metgen.py

```diff
@@ -536,21 +552,22 @@ def collection_from_cmr(environment: str, auth_id: str, version: int):
 
     # FYI: data format (e.g. NetCDF) is available in the umm-c response in
     # ArchiveAndDistributionInformation should we decide to use it.
-    spatial_extent = ummc_content(ummc, "SpatialExtent")
-    granule_spatial_rep = ummc_content(spatial_extent, constants.GRANULE_SPATIAL_REP)
-
-    # return error if no GranuleSpatialRepresentation
-    if not granule_spatial_rep:
-        raise Exception(
-            f"{constants.GRANULE_SPATIAL_REP} not available in UMM-C metadata for {auth_id}.{version}."
-        )
-
     return Collection(
         auth_id,
         version,
-        granule_spatial_representation=granule_spatial_rep,
-        spatial_extent=spatial_extent,
-        temporal_extent=ummc_content(ummc, "TemporalExtents"),
+        granule_spatial_representation=ummc_content(
+            ummc, ["SpatialExtent", constants.GRANULE_SPATIAL_REP]
+        ),
+        spatial_extent=ummc_content(
+            ummc,
+            [
+                "SpatialExtent",
+                "HorizontalSpatialDomain",
+                "Geometry",
+                "BoundingRectangles",
+            ],
```

Comment:
It might be useful to put these paths in as constants. Something like

UMMC_GRANULE_SPATIAL_REP_PATH = ["SpatialExtent", constants.GRANULE_SPATIAL_REP]
UMMC_BOUNDING_RECTANGLES_PATH = [
                "SpatialExtent",
                "HorizontalSpatialDomain",
                "Geometry",
                "BoundingRectangles",
            ]